interface-definitions/policy-local-route.xml.in

```diff
@@ -88,6 +88,14 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <leafNode name="incoming-interface">
```

Comment:
Please use `inbound-interface` like we have for NAT.

```xml
              <leafNode name="inbound-interface">
                <properties>
                  <help>Inbound interface</help>
                  <completionHelp>
                    <script>${vyos_completion_dir}/list_interfaces.py</script>
                  </completionHelp>
                </properties>
              </leafNode>
```

Using the same XML definition two times is discouraged. Please create a .xml.i file and use #include